conjure-go-contract/errors/error_code.go

```diff
@@ -152,6 +158,8 @@ func (ec *ErrorCode) UnmarshalText(data []byte) error {
 		*ec = CustomClient
 	case "CUSTOM_SERVER":
 		*ec = CustomServer
+	case "SERVICE_UNAVAILABLE":
```

Comment:
Are we accepting that these can not be included in generated errors? Otherwise this needs to be added to the enum here: https://github.com/palantir/conjure/blob/3d0fec99877ca6dbe899d3faba43c4b45866e9da/conjure-api/src/main/conjure/conjure-api.yml#L46